libs/components/autonumeric/src/lib/modules/autonumeric/autonumeric.directive.ts

```diff
@@ -127,12 +133,16 @@ export class SkyAutonumericDirective
         this.isFirstChange = false;
         this.control!.markAsPristine();
       }
-    }
 
-    if (typeof value === 'number') {
-      this.autonumericInstance.set(value);
-    } else {
-      this.autonumericInstance.clear();
+      if (typeof value === 'number') {
+        if (this.skyAutonumericFormChangesUnformatted) {
+          this.autonumericInstance.setUnformatted(value);
+        } else {
+          this.autonumericInstance.set(value);
+        }
+      } else {
+        this.autonumericInstance.clear();
+      }
```

Comment:
This changes the behavior of this component. Was it required for the AG Grid changes?

Reply:
Yes. I added an internal input here so that we can set the value of the autocomplete without it being formatted. Without this change, if you attempt to set the value of the input with `autonumeric` it will automatically format the number. This caused issues when trying to set the field to `4` and then allowing for additional input. Before this change, it would automatically format the `4` to `$4.00` (or whatever the correct formatting was) and there was no way to stop this. Because we want to be able to trigger editing my typing a number and having that number start the input - this was problematic. 

My changes simply add the internal input to stop the formatting and then leave all other previous logic in place.